src/main/java/com/tstepnik/planner/domain/task/TaskDTO.java

```diff
@@ -0,0 +1,53 @@
+package com.tstepnik.planner.domain.task;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Size;
+
+public class TaskDTO {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotEmpty
+    @Size(min=3)
+    private String description;
```

Comment:
On fields like this (desc, which can get really long) we should also add `max` validator because DB by default can probably take max of 255 chars. You can test it and see yourself :)

Reply:
So how make descriptions possible longer like post on facebook ?

Reply:
https://stackoverflow.com/questions/3868096/jpa-how-do-i-persist-a-string-into-a-database-field-type-mysql-text